en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc

```diff
@@ -1,12 +1,18 @@
 = MuleSoft Composer Release Notes
-:page-revdate: June 21, 2023
+:page-revdate: June 26, 2023
 
 Use MuleSoft Composer to build connected experiences faster. You don't need to wait for development resources to unlock data, integrate apps, and automate business processes that create real customer value.
 
 New versions are automatically available once they are released; you don't need to perform any tasks to move to the new version.
 
 See xref:ms_composer_ms_release_notes_ki.adoc[Known Issues and Limitations for MuleSoft Composer] for a list of known issues and limitations by date.
 
+== June 26, 2023
+
+include::_partials/release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+3]
+
+* You can now define a variable as temporary storage in your flow and use it as a data pill. Defining a variable allows you to complete certain use cases like aggregating calculations and storing values. Variables are defined locally within a flow and can be used anywhere within that flow.
```

Comment:
See comments on line 339. Can you reuse this content?

Reply:
Sadly no.